libs/sdk-backend-bear/src/backend/workspace/styling/styling.ts

```diff
@@ -19,13 +23,19 @@ export class BearWorkspaceStyling implements IWorkspaceStylingService {
     public getTheme = async (): Promise<ITheme> => {
         const config = await this.authCall((sdk) => sdk.project.getProjectFeatureFlags(this.workspace));
 
-        const enabledByFeatureFlag = config[ENABLED_THEMING_FEATURE_FLAG_SETTINGS_KEY];
+        const platformEdition = config[SETTINGS_KEY.PLATFORM_EDITION];
+
+        if (!platformEdition || platformEdition === SETTINGS_KEY.PLATFORM_EDITION_VALUE_ENTERPRISE) {
```

Comment:
the logic should be oposite - if Enterprise -> go and load theme even the enabledByFeatureFlag is false